### PR TITLE
feat(accessibility): add alternative text to weather icons

### PR DESCRIPTION
**Pull request by ardasnturk**
- added `description` information from API to add alternative text to weather icons

SVA-868

### CODE REVIEW

File: src/components/weather/DailyWeather.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ type Props = {
   };
 };
 
-export const DailyWeather = ({ date, icon, temperatures }: Props) => {
+export const DailyWeather = ({ date, desctription, icon, temperatures }: Props) => {
```
Comment (donni106) [inline src/components/weather/DailyWeather.tsx:25]: typo in this file! `desctription` -> `description`

File: src/components/weather/DailyWeather.tsx [hunk 2]
```diff
@@ -35,7 +36,10 @@ export const DailyWeather = ({ date, icon, temperatures }: Props) => {
           <WrapperRow center>
             <View style={styles.iconContainer}>
               <Image
-                source={{ uri: `https://openweathermap.org/img/wn/${icon}@2x.png` }}
+                source={{
+                  uri: `https://openweathermap.org/img/wn/${icon}@2x.png`,
+                  captionText: desctription
```
Comment (donni106) [inline src/components/weather/DailyWeather.tsx:41]: please share an example of data coming from the api.

Comment (donni106) [inline src/components/weather/DailyWeather.tsx:41]: when VoiceOver was saying "grafik" before, what is the result now for some case?

**Review by donni106 — CHANGES_REQUESTED** [review]

File: src/components/weather/DailyWeather.tsx (hunk 1 above)
Comment (ardasnturk) [reply src/components/weather/DailyWeather.tsx:25]: Upps sorry 🤦‍♂️ I will update immediately.

File: src/components/weather/DailyWeather.tsx (hunk 2 above)
Comment (ardasnturk) [reply src/components/weather/DailyWeather.tsx:41]: Here are some examples of descriptions from the API:
```
Leichter Regen
Bedeckt
Mäßig bewölkt
Klarer Himmel
Ein paar Wolken
Überwiegend bewölkt

Comment (ardasnturk) [reply src/components/weather/DailyWeather.tsx:41]: As you can see from the sample code I have shared below, it first reads the description and then reads the (Bild) text that was previously added to the code. 

For example: `Überwiegend bewölkt Bild`

```
accessibilityLabel={`${sourceProp.captionText ? sourceProp.captionText : ''} ${consts.a11yLabel.image}`}
```
`consts.a11yLabel.image = (Bild)`

Comment (donni106) [reply src/components/weather/DailyWeather.tsx]: i understand! very good.

**Review by donni106 — APPROVED** [review]

**Pull request merged**